ConsoleApp1/ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,334 @@
+﻿namespace Console
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Collections;
+    using System.IO;
+    using System.Text;
+    class Program
+    {
+        
+        static void Main()
+        {           
+           
+  
+        }  
+
+
+    }
+
+    #region Menu
+    public static class Menu
+    {
+        public static void Login()
+        {
+
+        }
+
+        public static void MainMenu()
+        {
+
+        }
+
+        public static void OrderMenu()
+        {
+
+        }
+
+        public static void CustomerDetails()
+        {
+
+        }
+
+        public static void GetBill()
+        {
+
+        }
+    }
+
+    #endregion
+
+    #region FileInterFaces
+
+    public class FileIO
+    {
+        public FileIO(ref Shop shop, bool init)
+        {
+            if (!init)
```

Comment:
as I see this method can be split on 3 different methods - write to 3 different files